Source/Plugins/EditorModules/SceneView/Modules/SceneView.cs

```diff
@@ -77,6 +74,14 @@ private class CreateContextEntryTag
 		private MenuModelItem	nodeContextItemCopy		= null;
 		private MenuModelItem	nodeContextItemPaste	= null;
 
+		private SceneViewSettings userSettings = new SceneViewSettings();
```

Comment:
Style consistency: Use a double newline to separate members of different kinds if either of them is grouped into semantic blocks on their own.

(This was probably wrong before your change, but might as well address it while here)